example.yml

```diff
@@ -108,8 +117,6 @@ awards:
     issuer: Toastmasters International
     url:
     location: City of London, United Kingdom
-    highlights:
```

Comment:
This `highlights` fields still needs to be present with a null value in order for the template to compile.

```
highlights: 
```

Reply:
Sorry my bad. Will update it.